File: fib.go
package main

import (
  "fmt"
  "os"
  "strconv"
)

func fib(a int) int {
  if a < 2 {
    return a
  } else {
    return fib(a - 1) + fib(a - 2)
  }
}

func main() {
  n, _ := strconv.Atoi(os.Args[1])
  fmt.Println(fib(n))
}

File: fib.erl
-module(fib).
-export([start/0, fib/1]).

fib(0) -> 0;
fib(1) -> 1;
fib(N) -> fib(N-1) + fib(N-2).

start() ->
    io:format("~p~n", [fib:fib(35)]).
